packages/webcert/src/feature/certificate/Modals/AboutWebcertModalContent.tsx

```diff
@@ -2,21 +2,27 @@ import React from 'react'
 import { InfoBox } from '@frontend/common/src'
 import styled from 'styled-components'
 import WCDynamicLink from '../../../components/utils/WCDynamicLink'
+import { shallowEqual, useSelector } from 'react-redux'
+import { getConfig } from '../../../store/utils/utilsSelectors'
 
 const Wrapper = styled.div`
-  p, h4 {
-  padding-bottom: 10px;
+  p,
+  h4 {
+    padding-bottom: 10px;
+  }
 `
 
 const AboutWebcertModalContent: React.FC = () => {
+  const { version } = useSelector(getConfig, shallowEqual)
+
   return (
     <Wrapper>
       <InfoBox type={'observe'}>
         <WCDynamicLink linkKey={'ineraIntygsskola'} />
       </InfoBox>
       <p className={'iu-pt-400'}>Webcert är en tjänst som drivs av Inera AB.</p>
 
-      <p>Nuvarande version är 6.10.0.64</p>
+      <p>Nuvarande version är {version}.</p>
```

Comment:
Is there a test for this?

Reply:
Do we really need a test for rendering a value? Each test we introduce has a cost, does the code justify it?

Reply:
You could write a test checking that if the value exists in the state it is visible on the page when rendering the modal.